src/services/K8sHelmService.ts

```diff
@@ -255,10 +249,19 @@ export class K8sHelmService {
             computedValues = yaml.safeLoad(values.join('\n')) || {};
         }
 
+        const searchValue = (text: string, label: string) => {
+            const value = text.split(new RegExp(`^${label}:\s*(.+?)^[A-Z]{2,}`, 'gmus'))[1];
```

Comment:
what is `s` flag?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp

Reply:
flag `s`: Single line - Dot matches newline